### PR TITLE
Update unicode_cldr to 43.0

### PR DESCRIPTION
**Pull request by supechicken**

### CODE REVIEW

**Review by satmandu — COMMENTED** [review]
Does this change filelists in the manifest directory?

File: packages/unicode_cldr.rb [hunk 1]
```diff
@@ -2,27 +2,14 @@
 
 class Unicode_cldr < Package
   description 'Unicode Common Locale Data Repository'
-  homepage 'http://cldr.unicode.org/'
-  @_ver = '38.0'
+  homepage 'http://cldr.unicode.org'
```
Comment (uberhacker) [inline packages/unicode_cldr.rb:5]: Change to `https://cldr.unicode.org/`?

**Comment by supechicken** [comment]
> Does this change filelists in the manifest directory?

Does it have filelist before? (as I know only packages with non-prebuilt sources have filelist generated)

File: packages/unicode_cldr.rb (hunk 1 above)
Comment (supechicken) [reply packages/unicode_cldr.rb]: Done

**Comment by satmandu** [comment]
> > Does this change filelists in the manifest directory?
> 
> Does it have filelist before? (as I know only packages with non-prebuilt sources have filelist generated)

Didn't realize binaries weren't needed!

**Review by satmandu — APPROVED** [review]

**Pull request merged**